content/articles/google-workspace-service.markdown

```diff
@@ -1,11 +1,11 @@
 ---
-title: Google Apps Service
-excerpt: How to set up Google Apps DNS using DNSimple one-click service.
+title: Google Workspace (formerly G Suite) Service
```

Comment:
I've opened a parallel PR to update the service itself: https://github.com/dnsimple/dnsimple-services/pull/109.

Will also need to update the screenshots used in this article.